### DISCUSSION TITLE
Download old recording

### DISCUSSION THREAD

**Discussion by schumissimo63** [blakeblackshear/frigate#10451] [General]
Hello everyone, I had to redo my frigate config because it no longer worked with home assistant, I went through an LXC with docker compose. The problem is that I had a folder in recording that I was able to save but it is no longer recognized in event and I cannot download the clips. Is it possible to restore it? Thank you for your answers.

**Comment by NickM-27** [comment]
Unless you have the DB file there is no way to restore it in the UI

**Comment by schumissimo63** [comment]
Thank you very much, it worked, I'm closing.
